Server/bkr/server/model/scheduler.py

```diff
@@ -2647,7 +2647,7 @@ def provision(self):
             if 'no_default_harness_repo' not in install_options.ks_meta \
                     and not self.harness_repo():
                 raise ValueError('Failed to find repo for harness')
-        ks_keyword = install_options.ks_meta.get('ks_keyword', 'ks')
+        ks_keyword = install_options.ks_meta.get('ks_keyword', 'inst.ks')
```

Comment:
shouldn't both 'ks' and 'inst.ks' be looked for?

Reply:
https://docs.python.org/3/library/stdtypes.html?highlight=dict#dict.get